buildozer/targets/android.py

```diff
@@ -569,7 +564,7 @@ def _android_update_sdk(self, *sdkmanager_commands):
             sdkmanager_path = self.sdkmanager_path
             sdk_root = f"--sdk_root={android_sdk_dir}"
             command = f"{yes_command} | {sdkmanager_path} {sdk_root} --licenses"
-            self.buildozer.cmd(command, cwd=self.android_sdk_dir)
+            self.buildozer.cmd(command, cwd=self.android_sdk_dir, shell=True)
```

Comment:
i believe this one could be replaced using pipes to capture stdout and inject it in stdin of sdkmanager, but indeed our abstraction probably doesn't make that convenient.

Reply:
Yeah, that was a quick fix in order to see the CI running. I expect to remove it before the merge.

We have another `shell=True` on iOS, that needs some attention, but doesn't make use of anything that is injectable by the user via `buildozer.spec`, so is low-prio.

Reply:
Changed this one, but kept the one on iOS.